test-apps/base-tests/package.json

```diff
@@ -63,17 +64,17 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-fetch": "^8.1.2",
-    "ember-load-initializers": "^2.1.2",
-    "ember-modifier": "^4.1.0",
-    "ember-page-title": "^8.0.0",
-    "ember-qunit": "^8.0.2",
-    "ember-resolver": "^11.0.1",
+    "ember-load-initializers": "^3.0.1",
+    "ember-modifier": "^4.2.0",
+    "ember-page-title": "^8.2.3",
+    "ember-qunit": "^8.1.1",
+    "ember-resolver": "^13.1.0",
     "ember-source": "~5.3.0",
     "ember-source-channel-url": "^3.0.0",
     "ember-template-lint": "^6.0.0",
     "ember-try": "^3.0.0",
     "ember-welcome-page": "^7.0.2",
-    "eslint": "^9.15.0",
```

Comment:
this was a mistake on main/master since it's been too hard to keep CI green throughout this process (of de-tangling some goofy v1 magic)